App/Containers/AddCaptionScreen.js

```diff
@@ -20,7 +20,7 @@ class AddCaptionScreen extends React.Component {
           }} />
         </TextileHeaderButtons>
       ),
-      headerRight: params.submitEnabled && (
+      headerRight: (
```

Comment:
Allow sharing with no caption in the UI. Redux layer already supports it. cc: @sanderpick

Reply:
sweet. will it be safe for older versions if the api starts sending objects without a caption key?